fe/fe-core/src/main/java/com/starrocks/connector/ConnectorFactory.java

```diff
@@ -12,22 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
 package com.starrocks.connector;
 
-public interface ConnectorFactory {
+import com.starrocks.connector.config.ConnectorConfig;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.lang.reflect.Constructor;
+
+public class ConnectorFactory {
+    private static Logger LOG = LogManager.getLogger(ConnectorFactory.class);
+
     /**
      * create a connector instance
      *
      * @param context - encapsulate all information needed to create a connector
      * @return a connector instance
      */
-    Connector createConnector(ConnectorContext context);
+    public static Connector createConnector(ConnectorContext context) {
+        if (null == null || !ConnectorType.isSupport(context.getType())) {
+            return null;
+        }
 
-    /**
-     * a unique string represents a kinds of connector
-     *
-     * @return a string of connector name
-     */
-    String name();
+        ConnectorType connectorType = ConnectorType.from(context.getType());
+        Class<Connector> connectorClass = connectorType.getConnectorClass();
+        Class<ConnectorConfig> ctConfigClass = connectorType.getConfigClass();
+        try {
+            Constructor connectorConstructor =
+                    connectorClass.getDeclaredConstructor(new Class[] {ConnectorContext.class});
```

Comment:
```suggestion
                    connectorClass.getDeclaredConstructor(ConnectorContext.class);
```
I think that's enough. What do you think